db/organizer.js

```diff
@@ -1,16 +1,14 @@
-const {sequelize} = require("./sequelizer")
-const {DataTypes} =  require("sequelize")
-
-console.log(sequelize)
+const { sequelize } = require("./sequelizer")
+const { DataTypes } = require("sequelize")
 
 const Organizer = sequelize.define("Organizer", {
     id: {
         type: DataTypes.INTEGER,
         autoIncrement: true,
-        primaryKey: true    
+        primaryKey: true
     },
-    email: {type: DataTypes.STRING, allowNull: false, unique:true},
-    password: {type: DataTypes.STRING,  allowNull: false},
+    email: { type: DataTypes.STRING, allowNull: false, unique: true },
```

Comment:
Organizator musi to mieć? A logowanie przez RSa?

Reply:
Przy logowaniu przez RSa czy jakikolwiek inny serwis tworzone jest organizer z emailem z RSa i losowym ciągiem znaków jako hasłem. A wolę to mieć niż potem jak będziemy chcieli coś wypisać zastanawiać się czemu nie widać organizera XD